README.md

```diff
@@ -1,2 +1,5 @@
+> ## ⚠️ warning!
```

Comment:
We want this warning in the intro page for Telliot.
Some of our team members are not familiar with github so because of this I will also create another Gitbook space `https://docs.telliot..tellor.io` so the message in the Telliot intro could be something like:
```
> ## ⚠️ Note!
Telliot docs are synced from the project repository. 
To Edit visit:
[Telliot gitbook space](https://docs.telliot..tellor.io)
or
[Telliot github repo](https://github.com/tellor-io/telliot/docs)
```

Reply:
Ok, so where exactly I need to put this warning? Do I need to add an intro document in Telliot repo docs?

Reply:
Yes the note should be in the Telliot docs. I think in the main README.md which is usually the entry point for all docs. Not sure how would that look like in gitbook, so maybe best to add this note once we organise the docs a bit better

Reply:
I create a PR in Telliot repo @krasi-georgiev, so we can close this one if no longer needed.